Objects/Player/Player.gd

```diff
@@ -75,7 +75,12 @@ func get_input():
 		self.attacking = false
 
 func set_parts_sprite(texturePath: String):
+	if (texturePath == ""):
+		LootSprite.hide()
+		return
```

Comment:
The `.hide()` by itself didn't actually fix the bug -- it just allowed the game to keep running, but the console would still error out loading an empty path for a resource. It was also calling `.hide()` and then `.show()`, which makes no sense. This return will stop it from trying to load nothing when the `texturePath` is empty.